lib/CLI.js

```diff
@@ -367,16 +419,32 @@ ${packageData.name} v${packageData.version} \
 
   exitWithStatus(error, stats) {
     if (error) {
-      if (error.message) { logger.error(error.message); }
+      if (error.message) {
+        process.stdout.write(
```

Comment:
```suggestion
        process.stderr.write(
```

Reply:
Although it seems like a valid point, I would refrain from making logic changes as the part of this pull request. If this is a serious concern, let's raise a separate issue to adjust it. OK?

Reply:
According to this diff, this entire block is new in your branch?

Reply:
Oh, you are right, my bad. I think I left it for debugging purposes. Will revert to `logger.error()`.

Reply:
It appeared to be some debugging left-over. Removed in 7b15907.